src/common/components/hosts/tableUtils.tsx

```diff
@@ -80,14 +82,15 @@ export const hostnameColumn = (
 };
 
 export const roleColumn = (
+  t?: TFunction,
```

Comment:
why optional ?

Reply:
Because in OCM we don't have translations implemented